packages/core/bootstrap/src/lib/ws/reducer.ts

```diff
@@ -88,6 +92,8 @@ export const subscriptionsReducer = createReducer<SubscriptionsState>(
       const isSubscribing = state[key]?.subscribing
       state[key] = {
         active: false,
+        wasEverActive: false,
```

Comment:
Not sure if we should do it like this. For example, there could be a bug in our events pipeline that would trigger `subscribe` on an already subscribed channel.

So we shouldn't assume it's `unsubscribed=false` on the `subscribe` event.
In the same event, the subscription is active meaning `wasEverActive: true`, we just have a bug :)

That's why I think it's better here to just update the `subscribing` and `input` data, and don't update the rest.

We could then add a WARN log here if we find a mismatch, an invariant in the state which will signal we have a bug.